GeometryService/src/STMMaker.cc

```diff
@@ -360,6 +430,21 @@ namespace mu2e {
     _magnetTableTopExtraLength = _config.getDouble("stm.magnet.stand.topExtraLength");
     _magnetTableTopHalfHeight  = _config.getDouble("stm.magnet.stand.topHalfHeight");
     _magnetTableLegRadius      = _config.getDouble("stm.magnet.stand.legRadius");
+		////////absorber//wyq
+		_absorberBuild            = _config.getBool("stm.absorber.build");//wyq
```

Comment:
Is this absorber in all geometry versions? If not, should it default to false so old versions still run?